src/vm/util.cpp

```diff
@@ -1877,7 +1877,7 @@ size_t GetCacheSizePerLogicalCpu(BOOL bTrueSize)
         }
     }
 
-#if defined(_TARGET_AMD64_) || defined (_TARGET_X86_)
```

Comment:
Is there a reason why it is ok to delete this fallback for AMD64, but we still need it for x86?

Reply:
the only reason is because I have not tested on x86 so I am reluctant to remove it.